src/Mapper/Mapper.test.ts

```diff
@@ -279,20 +283,19 @@ describe("Mapper", () => {
       });
 
       it("normalizes nested related data", async () => {
-        const normalizeStub = stub(fakeRelatedService.serializer, "normalize").callThrough();
-        await mockSerializer.normalize(rawModelData);
-
+        const normalizeStub = stub(fakeRelatedService.mapper, "normalize").callThrough();
```

Comment:
@JZiemnicki This test is working now.  We needed to make the concrete mapper like we thought, but we were using the wrong model for it (the MockModel is the one that has the related field, not the relatedModel).